adal/automation/WinFormsAutomationApp/WinFormsAutomationApp.csproj

```diff
@@ -103,8 +103,8 @@
     <PackageReference Include="Legacy2CPSWorkaround" Version="1.0.0">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Newtonsoft.Json" Version="10.*">
-      <PrivateAssets>All</PrivateAssets>
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2">
```

Comment:
>11.0.2" [](start = 57, length = 7)

in the two other places you moved to v6. Any reason you are not using the same version everywhere?

Reply:
The two other places have a hard requirement for v6. this project does not so it can be the latest version